src/styles/blocks.scss/minute-arrow.scss

```diff
@@ -0,0 +1,33 @@
+.minute-arrow {
+  animation: rotating 3600s infinite linear;
+  transform-origin: center bottom;
+
+  @keyframes rotating {
+    from {
+      transform: rotate(0deg);
+    }
+
+    to {
+      transform: rotate(359deg);
+    }
+  }
+
+  &__top {
+    box-sizing: border-box;
+    width: 13px;
+    height: 13px;
+    border-top: 10px green;
+    border-left: 6px transparent solid;
+    border-right: 6px transparent solid;
+    border-bottom: 17px red solid;
```

Comment:
you use many same styles, create general classes to add styles that both arrows have, and than add modification to change only styles that difference